flags_test.go

```diff
@@ -213,3 +218,41 @@ func (t *FlagsTest) Maps() {
 	ExpectEq("", f.MountOptions["rw"])
 	ExpectEq("jacobsa", f.MountOptions["user"])
 }
+
+func (t *FlagsTest) ResolvePathInParentProcess() {
```

Comment:
Write unit tests covering all flows

empty env variable
no env variable
filepath is absolute path